src/vscode-dts/vscode.proposed.chatProvider.d.ts

```diff
@@ -3,115 +3,35 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// version: 2
+// version: 3
 
 declare module 'vscode' {
 
 	/**
-	 * The provider version of {@linkcode LanguageModelChatRequestOptions}
-	 */
+	* The provider version of {@linkcode LanguageModelChatRequestOptions}
+	*/
 	export interface LanguageModelChatRequestHandleOptions {
 
 		/**
 		 * What extension initiated the request to the language model
 		 */
 		readonly requestInitiator: string;
```

Comment:
Might be good to have a proposed API check when extensions try to use this and the `requiresAuthorization`.